src/Orchard.Web/Modules/Orchard.Taxonomies/Controllers/AdminLocalizedTaxonomyController.cs

```diff
@@ -1,5 +1,4 @@
-﻿using System.Web.Mvc;
-using System.Web.Routing;
+﻿using System.Web.Routing;
```

Comment:
Actually, I just realized I misread the class names, but with `AdminLocalizedTaxonomyController` deriving from `LocalizedTaxonomyController`, this whole thing can be made a lot easier with proper inheritance. All the suggestions need to be applied together for them to work.
```suggestion
﻿using System.Web.Mvc;
﻿﻿using System.Web.Routing;
```